src/components/member-profile/index.js

```diff
@@ -379,15 +391,33 @@ const Profile = (props) => {
               alt={fullName}
             />
             {isSuperUser && (
-              <div className={classNames.memberStatus}>
-                <img
-                  alt="info icon"
-                  src="icons/info.png"
-                  width={20}
-                  height={20}
-                />
-                <span>{memberStatusMessage}</span>
-              </div>
+              <>
+                <div
+                  className={classNames.memberStatus}
+                  onMouseEnter={() => setShowSettings(true)}
+                  onMouseLeave={() => setShowSettings(false)}
+                >
+                  <img
+                    alt="info icon"
+                    src="icons/info.png"
+                    width={20}
+                    height={20}
+                  />
+                  <span>{memberStatusMessage}</span>
+                  {dev && (
+                    <span style={{ marginLeft: '0.8rem' }}>
+                      {isSuperUser && isOptionKeyPressed && (
```

Comment:
if we are checking isSuperUser above do we need to check it here?

Reply:
fixed